vue/dynamicforms/src/components/table/definitions/rows.ts

```diff
@@ -10,11 +10,11 @@ export default class TableRows {
 
   visibilityHandler: (rowPk?: number) => boolean | { callback: (isVisible: boolean) => void, once?: boolean };
 
-  next: string | null;
+  next!: string | null;
```

Comment:
why? I don't see declaration in constructor -> it can potentially be undefined or null.